src/mapml-viewer.js

```diff
@@ -579,7 +650,11 @@ export class MapViewer extends HTMLElement {
   }
   _toggleControls() {
```

Comment:
I am wondering if _toggleControls is needed any longer?  Can't we simply do document.querySelector('mapml-viewer').controls = false?  It seems to be used by ContextMenu.

Reply:
I removed it.